app/services/api-base-service.js

```diff
@@ -0,0 +1,70 @@
+import Ember from "ember";
```

Comment:
is it me or does it look like the whole file is indented 🤔 ?
Not that it matters much

Reply:
It's properly indented :)

Reply:
ok, github was just trying to trick me : )